Percolator/Percolator.tla

```diff
@@ -25,30 +24,39 @@ VARIABLES client_ts
 \* prewrite.
 VARIABLES client_key
 
-\* $key_data[k]$ is the set of multi-version data timestamp of the key.
-\* Only start_ts.
+\* $key_data[k]$ is the set of multi-version data of the key.
+\* Since we don't care about the concrete value of data, a record of
+\* [ts: start_ts] is sufficient to represent one data version.
```

Comment:
IMO, I prefer using start_ts for better understand explicitly.

Reply:
Original: key_data[k] is a Nat.
Now: key_data[k] is [ts: Nat].

Reason for using a record with a sole field is explained below.